### ISSUE TITLE
When ticking a multi pitch it should not put the height or grade of the route into the first pitch field

### ISSUE THREAD

**Issue by brendanheywood** [theCrag/website#3147]
**What happened?**

If a route has a height, but does not have heights for each pitch, then it should not put the "first" height into the field for pitch 1.

https://www.thecrag.com/event/1758684165
 
![image](https://user-images.githubusercontent.com/187449/38778593-462baa8c-40ff-11e8-949f-129df797ba3a.png)

**What you expected:**

* [ ] just don't do anything if we don't have pitch heights
* [ ] same for pitch grades

**Comment by lordyavin** [comment]
+1 to fix this ASAP

**Comment by brendanheywood** [comment]
Just had a quick look at this thinking it was frontend js, but it's backend process data somewhere in the pitch parsing logic. I can see two simple business rules options:

1) if the pitches field says > 1 pitches, but there is only 1 height then don't supply any heights to the pitches
2) if the pitches fields says > 1 pitches, but there is a mis match between the number of pitches and the number of supplied heights, then don't add the heights to the pitch fields

So essentially if we have an edge case of a 5 pitch route, and 4 pitch heights have been added do we want the 4 we have on the form?

Also I've just tested and the same applies to pitch grades. Grade is probably more important as it impacts cpr, no grade is definitely better than a wrong grade. If we were only dealing with heights I'd lean towards 1), and if only grades then 2). I'd want the logic to be consistent, so overall I think I'd prefer the stricter 2) so the pitches must align exactly.
